### DISCUSSION TITLE
why no system.mutations table in clickhouse local?

### DISCUSSION THREAD

**Discussion by l1t1** [ClickHouse/ClickHouse#54953] [Q&A]
I query it and got error messages, and how to see the process of delete or update operation. 
```
localhost.localdomain :) select * from system.mutations;

SELECT *
FROM system.mutations

Query id: 85b37396-63ea-4832-b4f2-945cd40ab27c


0 rows in set. Elapsed: 0.030 sec.

Received exception:
Code: 60. DB::Exception: Table system.mutations doesn't exist. (UNKNOWN_TABLE)
```

**Comment by l1t1** [comment]
system.numbers is not exist too. 
```
localhost.localdomain :) select * from system.numbers(3);

Syntax error: failed at position 29 ('('):

select * from system.numbers(3);

Expected one of: token, Dot, UUID, alias, AS, identifier, FINAL, SAMPLE, table, table function, subquery or list of joined tables, array join, LEFT ARRAY JOIN, INNER, ARRAY JOIN, JOIN, PREWHERE, WHERE, GROUP BY, WITH, HAVING, WINDOW, ORDER BY, LIMIT, OFFSET, SETTINGS, UNION, EXCEPT, INTERSECT, INTO OUTFILE, FORMAT, end of query

localhost.localdomain :) select * from numbers(3);

SELECT *
FROM numbers(3)

Query id: 991e5f8a-e74a-4a32-a05a-aef1e69b1a1a

┌─number─┐
│      0 │
│      1 │
│      2 │
└────────┘

3 rows in set. Elapsed: 0.005 sec.
```

**Comment by l1t1** [comment]
sorry I wrote wrong syntax, system.numbers works
```
localhost.localdomain :) SELECT * FROM system.numbers LIMIT 1;

SELECT *
FROM system.numbers
LIMIT 1

Query id: fb7f90a6-c010-43d6-8dd8-89a1258d91c6

┌─number─┐
│      0 │
└────────┘

1 row in set. Elapsed: 0.007 sec.
```

**Comment by ardenwick** [comment]
I think clickhouse-local is just a tool for some simple tasks, it doesn't depend on any stored data, so `system.mutations` is not needed.

Quote from [doc](https://clickhouse.com/docs/en/operations/utilities/clickhouse-local#:~:text=While%20clickhouse%2Dlocal%20is%20a%20great%20tool%20for%20development%20and%20testing%20purposes%2C%20and%20for%20processing%20files%2C%20it%20is%20not%20suitable%20for%20serving%20end%20users%20or%20applications.):

> While clickhouse-local is a great tool for development and testing purposes, and for processing files, it is not suitable for serving end users or applications.

**Reply by UnamedRus** [reply]
Kind of, but you can create MergeTree tables in it and have mutations, so system tables should exist IMO
